core/src/sql/statements/define/user.rs

```diff
@@ -86,8 +88,12 @@ impl DefineUserStatement {
 		self.hash = passhash;
 	}
 
-	pub(crate) fn set_session(&mut self, session: Option<Duration>) {
-		self.session = session;
+	pub(crate) fn set_token_dur(&mut self, dur: Option<Duration>) {
```

Comment:
nit, but I would probably opt for `set_token_duration` for quick readability

Reply:
I actually thought about this at the time only to think about it again now and agree with you. I will change it 👍